src/components/Game.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useState } from 'react'
+import Board from './Board'
+
+//component
+const Game = () => {
+  const initialBoard = ['', '', '', '', '', '', '', '', '']
+  const [gameHistory, setGameHistory] = useState<GameState[]>([initialBoard])
+  const [isXNext, setIsXNext] = useState<boolean>(true)
+
+  /**
+   * pointer function for handling click events from square component
+   * adds current board state to history[] state
+   * @param squares
+   */
+  const addGameHistory = (squares: string[], sliderValue: number) => {
+    setIsXNext(!isXNext)
+    /**
+     * clear the rest of history if the current game state is at a
+     * previous state in history and a new move is made
+     */
+    setGameHistory([...gameHistory.slice(0, sliderValue + 1), squares])
+  }
+
+  return (
+    <div>
+      <div className="status">
+        <h2>{'Next Player: ' + (isXNext ? 'X' : 'O')}</h2>
+      </div>
+      <Board
+        gameHistory={gameHistory}
+        addGameHistory={addGameHistory}
+        currentTurn={isXNext ? 'X' : 'O'}
```

Comment:
you add complexity my renaming state and manipulating it as you prop drill. It makes more since to stick with either isXNext or currentTurn, and use that as a top-level hook